semantics/psc-trees-identifier.ads

```diff
@@ -26,6 +26,8 @@ package PSC.Trees.Identifier is
       Str : Strings.U_String;
    end record;
 
+   type Tree_Ptr is access all Tree'Class;
```

Comment:
This declaration should probably be removed, since it is redundant with the similar declaration at line 84 of psc-trees.ads.